frontend/javascripts/dashboard/dataset_view.js

```diff
@@ -188,105 +131,108 @@ class DatasetView extends React.PureComponent<Props, State> {
     );
   }
 
-  renderTable() {
+  function renderTable() {
+    const filteredDatasets = features().isDemoInstance
+      ? context.datasets.filter(
+          d => !d.isPublic || d.owningOrganization === props.user.organization,
```

Comment:
Is this correct? On the demo instance, the user is only supposed to see datasets which are either **not** public (?) OR which belong to their own organization? Shouldn't it be `d.isPublic`?

Reply:
This is a bit weird, I agree. The problem I was trying to solve is that when a new user arrives on wk.org, they are presented with a massive amount of non-descriptively labeled datasets of which none (probably) are their own. This hack was to filter all published datasets from other orgs out. They would still be available through the publication tab, tho.
Not sure what the best way to handle this would be.

Reply:
Ok, but wouldn't this be equivalent to the following?

```
d => d.owningOrganization === props.user.organization
```

The current boolean expression could render datasets which are not public and belong to a different organization, but I assume the back-end wouldn't send these datasets in the first place?

Reply:
Yes, you are right :D